betty/locale/config.py

```diff
@@ -0,0 +1,92 @@
+"""
+Provide localization configuration.
+"""
+
+from contextlib import suppress
+from typing import Self, cast
+
+from typing_extensions import override
+
+from betty.config import Configuration
+from betty.locale import Localizable, StaticTranslations, StaticTranslationsLocalizable
+from betty.serde.dump import VoidableDump, Dump, Void, minimize
+from betty.serde.load import Asserter, Assertions
+
+
+class StaticTranslationsLocalizableConfiguration(Configuration):
+    """
+    Provide configuration for a :py:class:`betty.locale.Localizable`.
+    """
+
+    UNDETERMINED = "und"
```

Comment:
Does Babel have an existing constant? If not, move this to the main Locale API as an enum we could later extend.